dash/examples/ex.04.memalloc/main.cpp

```diff
@@ -19,20 +19,22 @@ int main(int argc, char* argv[])
   auto myid = dash::myid();
   auto size = dash::size();
 
-  dash::Array< dash::GlobPtr<int> > arr(size);
+  using pointer_t = decltype(dash::memalloc<int>(size_t{}));
```

Comment:
Can that be replaced with `dash::GlobMemAllocPtr<int>`? In an example this looks cleaner.

Reply:
Must have overseen that. Done.